README.md

```diff
@@ -1,22 +1,40 @@
-## Dev Tools Super Power
+# Dev Tools Super Power
 
-The Developer tools that browsers provide you are super powerful and are a really good way to edit HTML and CSS and see immediately how those changes are rendered in the browser. We're going to be using the Chrome browser.
+## Problem Statement 
+By now, you're pretty familiar with HTML structural elements, and how they relate to each other on the page (great job on that quiz!). You might even feel pretty comfortable writing your own valid HTML. But most of the time as developers, we'll be jumping in on projects that are already in progress, which means that we'll need a way to view existing HTML and CSS, and potentially a way to play around with elements on live pages to see how we might potentially adjust things. Luckily for us, most browsers have those kinds of tools already built in, called `developer tools`, or `dev tools` for short. 
```

Comment:
* Don't want to back-ref the quiz in case we cut it.
* "Much of the time"
* pref. joining projects
* break sentence at progress,
* "We'll want"
* Lede, and closing sentence: In this lesson we'll look at the Chrome `dev tools`.